osal/src/bsp/generic-linux/src/bsp_console.c

```diff
@@ -30,39 +30,8 @@
 #include <sys/types.h>
 #include <sys/wait.h>
```

Comment:
Posix dependencies should be removed

Reply:
I don't mind removing these, I left them in because they were left in for the rtems version as well for some reason which is what I used to compare.
<img width="889" alt="Screen Shot 2020-08-24 at 6 22 47 PM" src="https://user-images.githubusercontent.com/12236267/91102298-e43c4900-e636-11ea-9a7f-c2de2f7b4e19.png">

Reply:
Thinking it over I think its best its removed. Its possible when rtems is built that they built it on a linux host which is how those are defined. But for rtems they are removed.